src/main/scala/amba/axi4/AsyncCrossing.scala

```diff
@@ -9,6 +9,11 @@ import freechips.rocketchip.tilelink._
 import freechips.rocketchip.subsystem.CrossingWrapper
 import freechips.rocketchip.util._
 
+/**
+  * Source(Client) side for AXI4 crossing clock domain
```

Comment:
there is no Client-Manager concept in AXI. Just Master/Slave(according to spec of AXI) is enough.

Reply:
I'm in favor of picking up client/manager terminology within RC for AXI as well. The other terms are offensive.

Reply:
See: https://developer.arm.com/documentation/ihi0022/e/AMBA-AXI3-and-AXI4-Protocol-Specification

Reply:
I am aware of the AXI spec's terminology. I disagree with it and think the new terminology is equally clear and non-controversial. The spec has no normative requirement on terminology use. Source / Client and Sink / Manager are valid and understandable regardless of protocol context.

Reply:
Basically the manager-client terminology was from https://ieeexplore.ieee.org/document/7851474, it is designed for coherency system.
> Manager-Client Pairing (MCP) as a unifying methodology for designing multi-tier coherence protocols by formally defining and limiting the interactions between levels within a coherence hierarchy to enable composition.

We can use master-slave terminology for TileLink see PR #2532. But for the manager-client terminology, it can only be applied to a coherency system, while AXI4 is not, even for ACE or CHI they are not designed based on the idea of MCP...